### DISCUSSION TITLE
Including taskfiles: don't namespace and optional including

### DISCUSSION THREAD

**Discussion by back-2-95** [go-task/task#595] [Q&A]
Currently including a taskfile will result namespacing for commands in that file. Example:

```
includes:
  foobar: ./tools/task/Taskfile.yml
```

Will have commands like`task foobar:something`.

Is it be possible to include taskfile and declare tasks to be without namespace?

Other thing:

Can including be optional? E.g. if VARIABLE=1  then include some taskfile.

**Comment by ghostsquad** [comment]
includes does support an `optional` flag. There's another discussion about this here: https://github.com/go-task/task/issues/273

I just weighed in on it. I think flattening, or removing the "namespace" prefix hurts readability.
